dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/gaen/JDBCGAENDataServiceImpl.java

```diff
@@ -115,7 +121,7 @@ public int getMaxExposedIdForKeyDate(
   @Override
   @Transactional(readOnly = true)
   public List<GaenKey> getSortedExposedForKeyDate(
```

Comment:
Is there any test (in PostgresGaenDataServiceTest.java for example) that checks that active keys are not yet released?

Reply:
https://github.com/DP-3T/dp3t-sdk-backend/blob/c01ac3a1bed241104660ebb4e065a725173faa72/dpppt-backend-sdk/dpppt-backend-sdk-data/src/test/java/org/dpppt/backend/sdk/data/gaen/PostgresGaenDataServiceTest.java#L197